### PR TITLE
[docs] Convert WinC AWS docs to Markdown

### PR DESCRIPTION
**Pull request by codyhoag**
The initial conversion from Google Doc &rarr; Markdown. I plan to followup with some wordsmithing at a later date.

### CODE REVIEW

File: docs/aws/aws-with-windows-server.md [hunk 1]
```diff
@@ -0,0 +1,618 @@
+# OpenShift Container Platform 4.4 on AWS with Microsoft Windows Server 2019/1809 Nodes
```
Comment (aravindhp) [inline docs/aws/aws-with-windows-server.md:1]: Please add these files under  to [tools/ansible/docs](https://github.com/openshift/windows-machine-config-bootstrapper/tree/master/tools/ansible/docs).

Comment (codyhoag) [reply docs/aws/aws-with-windows-server.md]: Sure thing; just moved them. :+1:

**Comment by codyhoag** [comment]
I will squash commits once reviews have finalized. I wanted to preserve the changes based off the original doc.

**Comment by aravindhp** [comment]
@codyhoag, @Anandnatraj has approved so please squash your commits. I will approve and get it merged.

**Comment by codyhoag** [comment]
@aravindhp They're now squashed. Thanks!

**Pull request merged**